out_of_tree_boards/boards/arm/mec172x_mtl_s/mec172x_mtl_s.dts

```diff
@@ -188,10 +188,10 @@
 
 &spi0 {
 	status = "okay";
+	compatible = "microchip,xec-qmspi-ldma";
```

Comment:
This is missing for mec172xmodular overlay

Reply:
Though this should have been there in zephyr/boards/arm/mec172modular/ maybe need to submit a patch into mainline